test/pool-cl/CLPoolManager.t.sol

```diff
@@ -44,6 +44,35 @@ contract CLPoolManagerTest is Test, NoIsolate, Deployers, TokenFixture, GasSnaps
     using LPFeeLibrary for uint24;
     using Hooks for bytes32;
 
+    error ContractSizeTooLarge(uint256 diff);
+
+    event Initialize(
+        PoolId indexed id,
+        Currency indexed currency0,
+        Currency indexed currency1,
+        IHooks hooks,
+        uint24 fee,
+        bytes32 parameters
+    );
+    event ModifyLiquidity(
+        PoolId indexed poolId,
+        address indexed sender,
+        int24 tickLower,
+        int24 tickUpper,
+        bytes32 salt,
+        int256 liquidityDelta
+    );
+    event Swap(
+        PoolId indexed poolId,
+        address indexed sender,
+        int128 amount0,
+        int128 amount1,
+        uint160 sqrtPriceX96,
+        uint128 liquidity,
+        int24 tick,
+        uint24 fee,
+        uint24 protocolFee
+    );
```

Comment:
It seems that the three events did not use.
This test contracts still use the ICLPoolManager.Initialize .......

Reply:
oh, it should have been deleted. I guess it got recovered by mistake when resolving conflicts. Thx for reminding